phoenix-core/src/it/java/org/apache/phoenix/end2end/UpgradeIT.java

```diff
@@ -38,6 +40,7 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import com.google.common.collect.Lists;
```

Comment:
I don't believe we have a policy as a community against using Guava.

Reply:
(As a member of the community, @swaroopak , you are of course free to propose such a policy, but that should be a discussion for the larger community to debate, rather than an issue for a single PR.)

Reply:
My bad, I didn't realize it was coming from Guava. :)